internal/controller/driver_controller.go

```diff
@@ -296,7 +300,7 @@ func (r *driverReconcile) LoadAndValidateDesiredState() error {
 	opConfig := csiv1a1.OperatorConfig{}
 	opConfig.Name = operatorConfigName
 	opConfig.Namespace = operatorNamespace
-	if err := r.Get(r.ctx, client.ObjectKeyFromObject(&opConfig), &opConfig); client.IgnoreNotFound(err) != nil {
+	if err := r.Get(r.ctx, client.ObjectKeyFromObject(&opConfig), &opConfig); err != nil {
```

Comment:
This change does not make sense, the operator config is optional. We need to reconcile even if it does not exists

Reply:
done.